uPortal-webapp/src/main/webapp/scripts/portal-analytics.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+"use strict";
+
+// Wrapped in an IIFE to remove the global scope of the functions
+(function () {
+    // Function that captures a click on an outbound link in Analytics.
+    var outboundClick = function(event) {
+        if ((event === undefined) || (event === null)) {
+            // Tried to process an outbound click, but there was no originating event
+            return;
+        }
+
+        // Both path and composedPath need to be checked due to browser support
+        var anchorForEvent = (event.path || (event.composedPath && event.composedPath()))[0].closest('a');
```

Comment:
unless IE support is needed, `let` or `const` would be preferred
```suggestion
        const anchorForEvent = (event.path || (event.composedPath && event.composedPath()))[0].closest('a');
```